site/snippets/templates/home/playground/header.php

```diff
@@ -23,15 +23,18 @@ class="playground-header-figure-wrapper"
 					<?= img($storyImage, [
 						'alt' => $storyImage->alt()->or('Panel screenshot for: ' . $story->title()),
 						'src' => [
-							'width' => 1280
+							'width' => 1520
 						],
 						'lazy' => false,
+						'sizes' => '(min-width: 1760px) 1520px, (min-width: 800px) 85vw, 100vw',
 						'srcset' => [
 							320,
 							640,
+							800,
 							960,
-							1280,
-							1600
+							1520,
+							1600,
+							3040
```

Comment:
`1520` and its 2x counterpart `3040` are the exact sizes for when the browser width is larger than our layout max-width.

The `1600` came in as a 2x counterpart of `800`. But you are right that it is probably not needed because the browser might as well choose `1520` then.

Maybe we should rather redistribute all other sizes except `1520` and `3040` with the goal to reduce large gaps.

Reply:
Sounds good to me. Also for the other changes. If we can keep those "must-have" sizes and then rather redistribute a few alternatives as "evenly" as possible.